Formula/mmseqs2.rb

```diff
@@ -24,9 +23,13 @@ class Mmseqs2 < Formula
 
   resource "documentation" do
     url "https://github.com/soedinglab/MMseqs2.wiki.git",
-        revision: "c77918c9cebb24075f3c102a73fb1d413017a1a5"
+        revision: "d53d8be3761ee625b0dcddda29b092bbd02244ef"
   end
 
+  # Patch for SIMDe on AppleClang11, remove for next release
+  # https://github.com/simd-everywhere/simde/commit/e5b5b7db97892a378fa6d2714b299791c4c5eb10
+  patch :DATA
```

Comment:
I meant to use the URL style patch instead of DATA in the formula

Reply:
I am not sure how to apply the patch (https://github.com/simd-everywhere/simde/commit/e5b5b7db97892a378fa6d2714b299791c4c5eb10.patch) to a file in a subfolder. I tried doing that, but there didn't seem to be a good way to manipulate the patch.

Reply:
The file `simde/simde-common.h` is at `lib/simde/simde/simde-common.h` in our code base.

Reply:
You need to do something like this:
```
patch do
  url "https://github.com/simd-everywhere/simde/commit/e5b5b7db97892a378fa6d2714b299791c4c5eb10.patch?full_index=1"
  sha256 "<sha256sum of the above patch"
end
````

Reply:
I get the error message `can't find file to patch at input line 15`...

Reply:
Sorry, I misunderstood. So you're saying that the patch predates the file move? Then commit the fix for `simde-common.h` in its new location and replace the `url` above with the new commit URL.

Reply:
I could define the patch as a resource, regex replace the path to match the subdirectory and then apply it as a string. I am not sure that is better than the current solution though.

Reply:
The patch is for an upstream library that MMseqs2 depends on. The compilation issue was fixed upstream and the patch I am linking to is the upstream fix. We bundle the library as a git subtree and I don't want to commit anything ontop of the subtree as that would make later merging very difficult.

Reply:
Can we keep this minimal solution? I don't think it makes sense to produce a URL based patch. The compilation issue on Mojave is already fixed upstream by updating the dependency.

I could load this whole patch: https://github.com/soedinglab/MMseqs2/commit/9d3ab794fb84c4044a95672805e994e135318fcf.patch 

However, that's multiple tens of thousands of lines of diffs instead of two lines.